.gitignore

```diff
@@ -27,6 +27,7 @@ node_modules/
 
 # Flamegraph
 *.html
+!crates/swc_html*/**/*.html
```

Comment:
Not sure where do we generate htmls files, so using it, ideally we should ignore html files where we generate them

Reply:
It was used some profiling like `cargo build --timings`